### ISSUE TITLE
how to use head controlled UI in Android application

### ISSUE THREAD

**Issue by esteimle** [RokidGlass/glass2-docs#105]
I have installed my android application on your device but cannot access any of the menu items.  How can I use the same head controlled cursor found in your documents app?

**Comment by sunchaonew** [comment]
You can refer to the last section of the documentation below to use the heads-control functions.
https://rokidglasses.gitee.io/glass2-docs/zh/2-sdk/8-imu-sdk/
Selecting the control effect is calculated by calculating whether the center of the screen coordinate is inside the control coordinate. At present, you need to realize it yourself

**Comment by esteimle** [comment]
Hmm this is not what I meant.  I do not want a list. I want to click
anywhere in the app like in this video https://youtu.be/9sIQJO90TeY?t=27
 You do not support this in your SDK correct?

Thank you

**Comment by sunchaonew** [comment]
Yes, this anchor selection effect needs to be implemented by myself as I said above, the SDK does not provide this function for the time being
The bottom of above document address have demo of how to achieve your own head control function, you can refer to the next
